src/content/structured/components/cards/guidance.mdx

```diff
@@ -126,7 +126,7 @@ The following section describes the different areas of content on a card.
   imageSrc={cardsFig1}
   imageAlt="A diagram showing the positioning of the different areas within a card. The areas are a heading area, an image area, a content area, an interaction area, and an expandable area."
   state="none"
-  caption="A card includes a heading area, an image area, a content area, an interaction area, and an expandable area."
+  caption="A card includes a heading area, an image area, a content area, an interaction area and an expandable area."
```

Comment:
```suggestion
  caption="A card includes a heading, image, content, interaction and an expandable area."
```
Just to make the sentence more concise 😊

Reply:
I'm not sure on this but I can see that they're all referred to areas throughout the card guidance, as well as on the image this caption relates to. I think it makes sense to leave it for consistency, but happy to see what others think.

Reply:
Would be good to have designer input on this maybe?
@GCHQ-Developer-094 @GCHQ-Designer-777